decidim-system/app/views/decidim/system/shared/_api_users.html.erb

```diff
@@ -0,0 +1,65 @@
+<table class="stack">
+  <thead>
+    <tr><th><%= t("models.api_user.fields.organization", scope: "decidim.system") %></th>
+      <th><%= t("models.api_user.fields.name", scope: "decidim.system") %></th>
+      <th><%= t("models.api_user.fields.key", scope: "decidim.system") %></th>
+      <th><%= t("models.api_user.fields.secret", scope: "decidim.system") %></th>
+      <th><%= t("models.api_user.fields.created_at", scope: "decidim.system") %></th>
+      <th><%= t("actions.title", scope: "decidim.system") %></th>
+    </tr>
+  </thead>
+  <tbody>
+    <% api_users&.each do |api_user| %>
+      <tr>
+        <td>
+          <%= api_user.organization.host %>
+        </td>
+        <td>
+          <%= api_user.name %>
+        </td>
+        <td>
+          <%= translated_attribute(api_user.api_key) %>
+        </td>
+        <td>
+          <% if fresh_token?(api_user) %>
+            <div class="api-user-secret  flex flewx-row items-center gap-x-2">
+              <div class="input-group__password">
+                <input type="password" id="<%= "token_#{api_user.id}"%>" value="<%= params[:token] %>" class="w-full">
```

Comment:
The show button is not displayed after creating a new user:
![Show button not displayed](https://github.com/user-attachments/assets/810ed0fa-4d1e-476f-898d-7ccf436247c3)